tests/e2e/pageobjects/dashboard/Dashboard.ts

```diff
@@ -58,11 +61,37 @@ export class Dashboard {
         await this.workspaces.waitWorkspaceListItemAbcence(workspaceName);
     }
 
+     async stopAndRemoveWorkspaceWithSelectingTab(workspaceName: string) {
+        Logger.debug(`Dashboard.stopAndRemoveWorkspaceByUI "${workspaceName}"`);
+        await this.openDashboardTab()
+        await this.stopWorkspaceByUI(workspaceName);
+        await this.workspaces.deleteWorkspaceByActionsButton(workspaceName);
+        await this.workspaces.waitWorkspaceListItemAbcence(workspaceName);
+    }
+
     async openDashboard() {
         Logger.debug('Dashboard.openDashboard');
+        //sometimes we can have opened files. In this case we get Alert dialog. We should perform it.
         await this.driverHelper.getDriver().navigate().to(TestConstants.TS_SELENIUM_BASE_URL);
+        try {
+            await this.waitPage();
+        } catch (err) {
+            if (err instanceof error.UnexpectedAlertOpenError) {
+                this.driverHelper.getDriver().switchTo().alert().accept();
```

Comment:
It's possible to filter on particular alert text (`this.driverHelper.getDriver().switchTo().alert().getText()`).

It will allow to throw exception in case not unexpected error alert.

Reply:
So, which alert text is acceptable in here?
What about the next verification:
```
           if (err instanceof error.UnexpectedAlertOpenError) {
                alertText = await this.driverHelper.getDriver().switchTo().alert().getText()

                if (alertText == EXPECTED_ALERT_TEXT) {
                     this.driverHelper.getDriver().switchTo().alert().accept();
                } else {
                     throw new Error(`Unexpected alert: ${alertText} ${err}`);
                }
          }

```

Reply:
I don't see any sense to check the text. It just increased the complexity of the code and add extra logic. In this method, we have unambiguous behavior got from the page or not and text does not matter. Moreover - if we successfully manage user preferences we can switch off this option and even remove this logic

Reply:
I see.

Let's add comment with explanation that it's a temporary solution, and it's going to be rewritten with removal of workspace through API.

Reply:
So looks like we have the solution with config map I reverted the method